src/tools/genpolicy/src/policy.rs

```diff
@@ -408,9 +408,22 @@ impl AgentPolicy {
             ));
         }
 
+        // enable debug mode if specified by the user
+        let mut request_defaults = self.infra_policy.request_defaults.clone();
+        if self.config.debug_mode {
+            // append to array /bin/bash and /bin/sh
```

Comment:
In my opinion, "debug mode" should allow customer to perform most if not all K8S operations on the running pod, not just shell command, and not just exec related flows.